### ISSUE TITLE
Set :fire: to everything related to overviews

### ISSUE THREAD

**Issue by enekid** [CartoDB/cartodb#14271]
Context: [[PROJECT] Deprecate raster overviews](https://github.com/CartoDB/cartodb-management/issues/5193)

Note: Don't merge into master until authorized.

Don't create new overviews tables when importing large datasets.

**Comment by dgaubert** [comment]
PR: #14304

**Comment by javitonino** [comment]
I just realized, we probably want to get rid of extension stuff as well for a fully executed fatality.

**Comment by dgaubert** [comment]
I have mixed feeling about that. My first thought was about to get rid of the extension's functions, then I realized that those functions might be helpful in some scenarios. Furthermore, thinking about `cartodb-extension` as a piece of software it might have the sense to expose some functionality that we stopped of using it, I don't know if there are people using the extension directly.

In any case, I don't have a strong opinion about that. I'm totally open to another point of view.
